site/content/release-guide.md

```diff
@@ -188,18 +198,55 @@ cd polaris-dist-dev
 mkdir x.y.z
 cp /path/to/polaris/github/clone/repo/build/distribution/* x.y.z
 cp /path/to/polaris/github/clone/repo/runtime/distribution/build/distributions/* x.y.z
-cp -r /path/to/polaris/github/clone/repo/helm/polaris helm-chart/x.y.z 
 svn add x.y.z
+```
+
+### Stage Helm Chart package
+
+You can now create a Helm package with the following command in the Polaris source folder:
+
+```
+cd helm
+helm package --sign --key '<YOUR KEY ID HERE>' --keyring ~/.gnupg/secring.kbx polaris
+```
+
+Create the signature and checksum for the Helm package tgz and prov files:
+
+```
+shasum -a 512 polaris-x.y.z.tgz > polaris-x.y.z.tgz.sha512
+gpg --armor --output polaris-x.y.z.tgz.asc --detach-sig polaris-x.y.z.tgz
+shasum -a 512 polaris-x.y.z.tgz.prov > polaris-x.y.z.tgz.prov
+gpg --armor --output polaris-x.y.z.tgz.prov.asc --detach-sig polaris-x.y.z.tgz.prov
+```
+
+Copy and Add the Helm package files to dist folder:
+
+```
+cd polaris-dist-dev
+mkdir helm-chart/x.y.z
+cp helm/*.tgz*  helm-chart/x.y.z
 svn add helm-chart/x.y.z
-svn commit -m"Stage Apache Polaris x.y.z RCx"
+```
+
+You can now update the Helm index:
+
+```
+cd helm-chart
+helm repo index .
```

Comment:
Indexing is also required after the we move it to the release repo, as the links are different.

Reply:
Actually, I think we should remove index on dist dev and only do that on dist release. I will update the release guide in that way (for discussion).

Reply:
Good point. The index in staging is only useful for testing RC (as it's not the one used after release). Let me update here.